server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QGatesWs.java

```diff
@@ -79,8 +79,8 @@ static void addConditionParams(NewAction action) {
       .setPossibleValues(QualityGateConditionDto.ALL_OPERATORS);
 
     action.createParam(PARAM_PERIOD)
-      .setDescription("Condition period")
-      .setExampleValue("1");
+      .setDescription("Condition period. If not set, the absolute value is considered.")
```

Comment:
Value is null or "1". Could the parameter be deprecated and replaced by a boolean property "onLeakPeriod" ?